rfc/system/0007-community-groups.md

```diff
@@ -0,0 +1,120 @@
+
+SIGs are voluntary groups which operate in the open and anyone can join.
+
+## Working Groups
+
+Designed to facilitate discussions/work regarding topics that are short-lived or that span multiple SIGs, Working Groups are distinct from SIGs in that they are intended to:
+
+- facilitate collaboration across SIGs
+- facilitate an exploration of a problem / solution through a group with minimal governmental overhead
+
+### It is a Working Group if...
+
+- It has a clear goal measured through a specific deliverable or deliverables
+- It is temporary in nature and will be disbanded after reaching its stated goal(s)
+
+## Processes
+
+### How to start and operate a SIG
+- [ ] Define and submit a charter including:
+  - [ ] Scope. It should be a short statement describing SIG's goal at a high level
+  - [ ] List of technical assets owned by this SIG (code repos, etc)
+  - [ ] Non-goals or Out of scope items
+  - [ ] Roles
+    - [ ] SIG Chair(s)
+        - Should be at least "Community members" as defined in GOVERNANCE
+        - Should define how priorities and commitments are managed
+        - Leads taking an extended leave of 1 or more months SHOULD coordinate with other leads to ensure the role is adequately staffed during their leave
+        - Leads MAY decide to step down at anytime and propose a replacement.
+   - [ ] Communication channels
+       - [ ] Meetings should occur at least once per month. Use [Doodle](https://doodle.com/en/) to coordinate if needed
+           - Meetings should be recorded and notes public
+       - [ ] Slack channel to use or create
+
+- [ ] Submit a PR to [/community](https://github.com/flyteorg/community) with the charter and request reviews from [Technical Steering Committee](https://github.com/flyteorg/community/blob/main/MAINTAINERS.md) Members
+-  A new SIG is approved by a supermajority of TSC active members
+
+### How to start and operate a WG
+
+- [ ] Define and submit a charter including:
+    - [ ] The problem this WG is trying to solve
+    - [ ] The artifact(s) it will deliver
+    - [ ] The SIG(s) it will collaborate with
+    - [ ] Repo ownership (if any)
+- [ ] Roles
+    - [ ] Lead(s)
+        - [ ] Should be at least “Community members” as defined in GOVERNANCE
+        - [ ] Must coordinate projects updates to the wider community.
+- [ ] Submit a PR to [/community](https://github.com/flyteorg/community) with the charter and request reviews from [Technical Steering Committee](https://github.com/flyteorg/community/blob/main/MAINTAINERS.md) Members
+-  A new WG is approved by a supermajority of TSC active members
+
+## Community groups retirement process
+
+### SIGs
+In the event that the SIG is unable to regularly establish consistent quorum or otherwise fulfill its responsibilities
+
+- after 3 or more months it SHOULD be retired
+- after 6 or more months it MUST be retired
+
+### WGs
+
+Working Groups will be disbanded if either of the following is true:
```

Comment:
Should we include something like "the mission of the WG is accomplished", as it should be very often the case for a WG? Or is this only intended to capture "dangling" WGs.

Reply:
Agree, I think I forgot to have a positive mindset :)